src/org/thoughtcrime/securesms/database/SmsDatabase.java

```diff
@@ -501,6 +504,20 @@ private void removeReadReminder(String where, String[] arguments){
     notifyConversationListeners(threadId);
   }
 
+  public void updateMessageBody(MasterSecretUnion masterSecret, long messageId, String body) {
```

Comment:
So this is permanently changing the body of the message in the database?

Reply:
Yes, I set the message as "${translatedMessage} [${untranslatedMessage}]"